x/rollapp/keeper/msg_server_update_state_test.go

```diff
@@ -147,55 +146,31 @@ func (suite *RollappTestSuite) TestUpdateState() {
 		suite.Require().Nil(err)
 
 		// end block
-		responseEndBlock := suite.App.EndBlocker(suite.Ctx, abci.RequestEndBlock{Height: suite.Ctx.BlockHeight()})
+		suite.App.EndBlocker(suite.Ctx, abci.RequestEndBlock{Height: suite.Ctx.BlockHeight()})
+
+		for i := uint64(0); i <= expectedLatestStateInfoIndex.Index; i++ {
+			expectedStateInfo, found := suite.App.RollappKeeper.GetStateInfo(suite.Ctx, rollapp.GetRollappId(), i)
```

Comment:
same here with the `expected` var name.
also because it's an index which is incremented by 1, is there a reason it shouldn't be found?
If not, I'd add `suite.Require().EqualTrue(found)`

Reply:
found removed